6_2_C.py

```diff
@@ -0,0 +1,15 @@
+from timeit import default_timer as timer
+
+
+def my_timer(f, *params):
+    """
+    :param f: function parameters

```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names